README.md

```diff
@@ -38,6 +38,10 @@ Note: **Do not use `npm ci`** on this project, as it will remove the Flyway libr
 ## Setting environment variables
 Create a `.env` file and populate it. The `.env-example` file provides a template for this.
 If a variable in this file is also set on the system, the file variable _will not_ overwrite the system variable.
+You will have to decide what you want your `DB_NAME`, `DB_USER` and `DB_PW` to be. 
+If you are running on your local machine, the `DB_HOST` will be `localhost`.
+The `FPR_FULL` variable can be `/scratch2/groups/seqware/nsdev/backups/seqware_files_report_latest.tsv.gz`, and the `[...temporaryLocation...]` blocks can be anywhere on your computer.
```

Comment:
Please remove the `FPR_FULL` hardcoded path, as this will vary from installation to installation and won't apply to any installations outside of OICR.